doc/src/appendices/appendices-master.rst

```diff
@@ -0,0 +1,42 @@
+.. Appendices Label:
+
+**********
+Appendices
+**********
+
+.. toctree::
+   :maxdepth: 2
+
+   suiterc-config-ref
+   site-user-config-ref
+   gcylc-config-ref
+   gscan-config-ref
+   remote-job-management
+   command-ref
+   gcylc-graph-view
+   readme-file
+   install-file
+   dev-history-major-changes
+   cylc-6-migration-ref
+   known-issues
+   licensing
+
+
+.. insert vertical whitespace else sidebar menu overhangs short page (ugly)
```

Comment:
This hack might work for the HTML version (provided that a particular theme is used), however, if we change the theme or build to a different format (e.g. PDF, slides)...

We could probably fix this in the chosen theme either in the Sphinx repository or with a custom CSS file (which would only affect that theme).

Reply:
[Addressing in latest comment; see :high_brightness:.]